src/panels/config/integrations/ha-config-integration-page.ts

```diff
@@ -338,6 +341,36 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
                 ${this._manifest?.version != null
                   ? html`<div class="version">${this._manifest.version}</div>`
                   : nothing}
+                ${this._manifest?.quality_scale &&
+                this._manifest?.quality_scale !== "internal"
+                  ? html`<ha-alert
+                      class=${classMap({
+                        [`${this._manifest.quality_scale}-medal`]: true,
+                      })}
+                      alert-type="info"
+                      ><ha-svg-icon
+                        class="medal"
+                        slot="icon"
+                        path=${this._manifest.quality_scale === "platinum"
+                          ? mdiTrophy
+                          : mdiMedal}
+                      ></ha-svg-icon>
+                      ${this.hass.localize(
+                        `ui.panel.config.integrations.config_entry.${this._manifest.quality_scale}_quality`
+                      )}
+                      <a
+                        slot="action"
+                        href=${`https://developers.home-assistant.io/docs/core/integration-quality-scale/#-${this._manifest.quality_scale}`}
```

Comment:
We need to link to the end-user version from our UI:

<https://www.home-assistant.io/docs/quality_scale/>

(that page will be updated for the final release).

../Frenck

Reply:
I'll use it, makes the code clearer